client/my-sites/upgrades/trials/start-trial/free-trial-confirmation-box.js

```diff
@@ -35,6 +35,15 @@ const FreeTrialConfirmationBox = React.createClass( {
 				<TermsOfService />
 				<div className="payment-box-actions">
 					<PayButton
+						beforeSubmitText={
+							this.translate( 'Start %(days)s Day Free Trial', {
```

Comment:
Does this need to pass 14 in the args like this? I think the reason for doing that is if this number will change programatically, which this one won't...